physics/ozphys_time_vary.F90

```diff
@@ -0,0 +1,165 @@
+! ###########################################################################################
```

Comment:
I'm curious about the decision to put this in its own scheme rather than keep this in GFS_phys_time_vary? Doesn't this just mean longer SDFs and that every SDF that uses ozphys will need to be edited?

Reply:
I chose to organize it this way for clarity, at the cost of SDF brevity. 

In general, how do we handle the situation when a scheme has two steps, separated by other schemes? 
Often we just add scheme specific _pre and _post routines, which we could move up or down the SDF away from the scheme. 

The ozone physics has two parts, 1) Interpolate ozone concentration before calling radiation and 2) compute prognostic ozone within the physics. In the case when using climatological ozone, we only need the first part to couple to the radiation.

In the case of the ozone physics, there is a _pre step to setup the o3 concentrations, but it was buried in an interstitial driver for "physics time-varying", whatever that means. Here I made that explicit, and hopefully more interoperable. Also, it puts this scheme on equal footing wrt to how other schemes are organized.